### ISSUE TITLE
Workspace log proto uses Python reserved keyword

### ISSUE THREAD

**Issue by Utsav2** [bazelbuild/bazel#6887]
> ATTENTION! Please read and follow:
> - if this is a _question_ about how to build / test / query / deploy using Bazel, ask it on StackOverflow instead: https://stackoverflow.com/questions/tagged/bazel
> - if this is a _discussion starter_, send it to bazel-discuss@googlegroups.com
> - if this is a _bug_ or _feature request_, fill the form below as best as you can.

### Description of the problem / feature request:

Workspace log proto uses Python reserved keyword

### Feature requests: what underlying problem are you trying to solve with this feature?

The protobuf field here is named `from`: https://source.bazel.build/bazel/+/master:src/main/java/com/google/devtools/build/lib/bazel/debug/workspace_log.proto;l=84?q=WorkspaceEvent. 

```
message SymlinkEvent {
  // path to which the symlink will point to
  string from = 1;
  // path of the symlink
  string to = 2;
}
```

Since `from` is a keyword, Python protobuf users can't use this field.

```
python
Python 2.7.12 (default, Nov 12 2018, 14:36:49) 
[GCC 5.4.0 20160609] on linux2
Type "help", "copyright", "credits" or "license" for more information.
>>> a = object()
>>> a.from = object()
  File "<stdin>", line 1
    a.from = object()
         ^
SyntaxError: invalid syntax
```

See https://stackoverflow.com/questions/30142750/reserved-keyword-is-used-in-protobuf-in-python for more context.

Could these be renamed to `from_path` and `to_path`, and the old uses be removed?

**Comment by benjaminp** [comment]
#6885

**Comment by Utsav2** [comment]
This was fixed by #6885
